game/scripts/npc/abilities/electrician_electric_shield_oaa.txt

```diff
@@ -27,47 +27,62 @@
-    "AbilityManaCost"                                     "5"
+    "AbilityManaCost"                                     "20"
 
     // Special
     //-------------------------------------------------------------------------------------------------------------
     "AbilityValues"
     {
+      "AbilityCooldown"
+      {
+        "value"                                                "0.5"
+        "special_bonus_facet_electrician_alternating_current"  "=16"
+      }
+      // max mana pool % to convert into shield at a time
       "mana_cost"                                         "20"
-      "shield_per_mana"                                   "0.75 0.9 1.05 1.2 1.35 1.5"
+      "shield_per_mana"
+      {
+        "value"                                           "0.75 0.9 1.05 1.2 1.35 1.5"
+        "CalculateSpellDamageTooltip"                     "0"
+      }
       "shield_damage_block" // original: 50% at all levels
       {
         "value"                                           "10 18 26 34 42 50"
         "CalculateSpellDamageTooltip"                     "0"
       }
-      "shield_duration"                                   "12.0"
+      "shield_duration"
+      {
+        "value"                                                "12"
+        "special_bonus_facet_electrician_alternating_current"  "=6 =7 =8 =9 =11 =13"
+      }
       "aura_radius"
       {
         "value"                                           "350"
         "affected_by_aoe_increase"                        "1"
       }
       "aura_damage"
       {
-        "value"                                           "25 35 45 55 110 220"
-        "LinkedSpecialBonus"                              "special_bonus_electrician_electric_shield_damage"
+        "value"                                                           "25 35 45 55 110 220"
+        "special_bonus_unique_electrician_electric_shield_damage"         "+100"
+        "special_bonus_facet_electrician_alternating_current"             "20%"
       }
-      "aura_interval"                                     "0.25"
-      "shard_shield_per_mana"
+      "damage_pct_to_attacks"
       {
-        "value"                                           "0.75 0.9 1.05 1.2 1.35 1.5"
-        "RequiresShard"                                   "1"
+        "value"                                                "0"
+        "special_bonus_facet_electrician_alternating_current"  "=100"
       }
-      "shard_shield_damage_block"
+      "mana_pct_refund"
       {
-        "value"                                           "10 18 26 34 42 50"
-        "CalculateSpellDamageTooltip"                     "0"
+        "value"                                           "0"
         "RequiresShard"                                   "1"
+        "special_bonus_shard"                             "=90"
+      }
+      "aura_interval"                                     "0.25"
+      "is_toggleable"
```

Comment:
this being here is an artifact of it being part of a facet before, but might as well leave it here in case we want some other facet or something to disable the auto-cast feature later